gradle/libs.versions.toml

```diff
@@ -1,7 +1,7 @@
 [versions]
 accompanist = "0.28.0"
 androidDesugarJdkLibs = "1.2.0"
-androidGradlePlugin = "7.3.1"
+androidGradlePlugin = "8.0.0-alpha01"
```

Comment:
Is this a necessary change to start collecting baseline profiles automatically?

Reply:
There was a bug in GMD, which caused sometimes not enough time to boot the emulator. It was also cherry picked to 7.4.0-rc1, so you might use that as well